SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1101,6 +1103,25 @@ void USpatialReceiver::OnComponentUpdate(const Worker_ComponentUpdateOp& Op)
 	case SpatialConstants::NETMULTICAST_RPCS_COMPONENT_ID:
 		HandleRPC(Op);
 		return;
+	case SpatialConstants::TOMBSTONE_COMPONENT_ID:
+		if (const Tombstone* TombstoneComponent = StaticComponentView->GetComponentData<Tombstone>(Op.entity_id))
+		{
+			if (TombstoneComponent->bIsDead)
+			{
+				RemoveActor(Op.entity_id);
+				return;
+			}
+		}
+	}
+
+	// If this entity has a Tombstone component marked dead, abort all component processing
```

Comment:
I don't think we can guarantee that no other component updates will come in for a particular entity after the Tombstone update, so we may need to guard against it (ex. if some other component is updated in the same oplist).

Reply:
It would be nice to catch this earlier (as early as possible) so we don't hvae to check tombstone status on every component update.  Any thoughts on storing a fast lookup container of deleted entity ids higher up the callstack, ex. the dispatcher, which is used to drop ops at that level?

Reply:
i've added an editor-only oplist now on the netdriver to make sure PIE cleans up properly on shutdown.  We could use that at runtime as well if you think it's appropriate.